packages/forms/src/directives/validators.ts

```diff
@@ -579,8 +583,12 @@ export class MinLengthValidator implements Validator, OnChanges {
   }
 
   private _createValidator(): void {
-    this._validator = minLengthValidator(
-        typeof this.minlength === 'number' ? this.minlength : parseInt(this.minlength, 10));
+    this._validator =
+        this.enabled() ? minLengthValidator(toInteger(this.minlength!)) : nullValidator;
+  }
+
+  enabled(): boolean {
```

Comment:
We'd need to avoid the `enabled` function from showing up in the `forms.d.ts` file (thus becoming a public API) since the `enabled` function is a part of the internal implementation. We can not mark it as `private` modifier unfortunately since it's used in host binding and it would fail in ViewEngine. However we can use `@internal` annotation (similarly let's add this annotation in the maxlength validator):
```suggestion
  /** @internal */
  enabled(): boolean {
```

Reply:
strangely after adding internal it's failing in test case saying
`packages/examples/forms/ts/ngModelGroup/ng_model_group_example.ts.NgModelGroupComp.html(6,9): Directive MinLengthValidator, Property 'enabled' does not exist on type 'MinLengthValidator'.`

And what I noticed in all other files when it's meant to be used internally only, then it starts with `_` so thinking to rename it to `_enabled()` let me know your thought.

Reply:
I believe that internal means that it will not appear in the .d.ts typings file for forms, so cannot be used outside the package.

Reply:
yes makes sense, I will get it removed. And update method name to `_enabled` thanks.

Reply:
I think there is a `@nodoc` tag too?

Reply:
yes after your message and on analysing file, I found we are using internal in case of properties and nodoc in case of method.

But FYI this this is not case of every file.
If you want me to update it to nodoc then let me know.

Reply:
@AndrewKushnir any input for test case failure scenario

Reply:
Yeah, it looks like the `test` CI job is failing because of this change. Let's use `@nodoc` for now then and we can change that to private later once ViewEngine is removed.

Reply:
@AndrewKushnir I have done required changes, if you have any other input than is it possible to share by today ?

As I am planning to work on second change of moving enabled function to `AbstractValidatorDirective` and it  will help me in it.